src/sui/sui_side_bar.c

```diff
@@ -105,6 +107,16 @@ static gint list_sort_func(GtkListBoxRow *row1, GtkListBoxRow *row2,
     g_return_val_if_fail(GTK_IS_EVENT_BOX(event_box), 0);
     item2 = SUI_SIDE_BAR_ITEM(gtk_bin_get_child(GTK_BIN(event_box)));
 
+    win = SUI_WINDOW(gtk_widget_get_toplevel(event_box));
+    cfg = sui_window_get_config(win);
+
+    if (g_strcmp0(cfg->sidebar_sort, "alphabetical") == 0){
```

Comment:
it would be better to define "alphabetical" and "recent" as macro in the header file.